mysql/util.go

```diff
@@ -130,10 +130,13 @@ func AppendLengthEncodedInteger(b []byte, n uint64) []byte {
 
 func RandomBuf(size int) []byte {
 	buf := make([]byte, size)
-	mrand.Seed(time.Now().UTC().UnixNano())
+	// When this project supports golang 1.20 as a minimum, then this mrand.New(...)
+	// line can be eliminated and the random number can be generated by simply
+	// calling mrand.Intn()
+	random := mrand.New(mrand.NewSource(time.Now().UTC().UnixNano()))
```

Comment:
Although this line is no longer needed with golang 1.20+ and simply the call to `mrand.Intn()` can be used, I've created a new random from a Source because this projects `go.mod` declares golang 1.18. But left a note in the code for future reference.